<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="fourtek-desktop" ignored="0" name="Manager" tests="6" failures="1" timestamp="18 Apr 2018 06:59:24 GMT" time="20.982" errors="0">
  <testcase name="doMangerLogin" time="3.018" classname="com.testcase.ManagerTest"/>
  <testcase name="selectTheCreatedProject" time="2.27" classname="com.testcase.ManagerTest"/>
  <testcase name="selectOrder" time="2.339" classname="com.testcase.ManagerTest"/>
  <testcase name="selectLanguageTranslator" time="1.644" classname="com.testcase.ManagerTest"/>
  <testcase name="assignProjectToTranslator" time="7.759" classname="com.testcase.ManagerTest">
    <failure type="java.lang.AssertionError" message="expected [Successfully assigned!] but found [×
Successfully assigned!]">
      <![CDATA[java.lang.AssertionError: expected [Successfully assigned!] but found [×Successfully assigned!]at com.testcase.ManagerTest.assignProjectToTranslator(ManagerTest.java:46)... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- assignProjectToTranslator -->
  <testcase name="doManagerLogout" time="3.934" classname="com.testcase.ManagerTest"/>
</testsuite> <!-- Manager -->
